portfolio/pom.xml

```diff
@@ -38,10 +38,36 @@
       <artifactId>jetty-annotations</artifactId>
       <version>${jetty.version}</version>
     </dependency>
-  </dependencies>
 
+    <!--Translate library-->
+    <dependency>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>google-cloud-translate</artifactId>
+  <version>1.70.0</version>
+</dependency>
+
+  
+</dependencies>
+
+ <!--
```

Comment:
Unless you really plan on using this later, I think you can just delete the commented out stuff.